README.md

```diff
@@ -546,6 +546,30 @@ ort:
     storageWriters: ["artifactoryStorage"]
 ```
 
+### AWS S3 Storage
+
+An AWS S3 Bucket can be used to store scan results. You must provide a previously created Bucket, an optional AWS region
```

Comment:
Could you make "AWS S3 Bucket" a link to https://docs.aws.amazon.com/AmazonS3/latest/userguide/Welcome.html? Will help ORT users quickly find the docs on how to set up a bucket.